docs/synapse/userguides/ug012_storm_ref_datamod.rst

```diff
@@ -256,7 +256,52 @@ None.
 delprop()
 ---------
 
-Todo
+Delete a property from the specified node(s).
+
+**Operator Syntax:**
+
+.. parsed-literal::
+  *<query>* **delprop(** *<prop>*, [ *force=<n>* ] **)**
+
+**Macro Syntax:**
+
+.. parsed-literal::
+  *<query>* **[ -:** *<prop>* ... **]**
+
+**Examples:**
+::
+  # Operator examples
+
+  inet:fqdn = vertex.link delprop(:created)
+
+  inet:fqdn = vertex.link delprop(:created, force=1)
+
+  # Macro examples
+
+  inet:fqdn = vertex.link [ -:created ]
+
+  inet:fqdn = vertex.link [ -:created -:updated ]
+
+**Usage Notes:**
+
+* ``delprop()`` operates on the output of a previous query.  It does not consume the tufos from the query.
+* Props which are read-only (``ro=1``) or have a default value (``defval``) on them cannot be deleted.
+
+**Operator Syntax Notes:**
+
+* The ``force`` keyword argument is required in order to delete props using the operator syntax.
```

Comment:
Should this read "The force keyword argument is required..." or "The force=1 keyword argument..." ?

Reply:
replaced with language from delnode()